### PR TITLE
Themes | Business Dashboard - added box-shadow theming

### PR DESCRIPTION
**Pull request by dimafeldman**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6263

File: src/Foundation/stylable/shadows.st.css [hunk 1]
```diff
@@ -6,9 +8,9 @@
...
   s6: rgba(22, 45, 61, 0.3);
   s7: rgba(22, 45, 61, 0.36);
-
-  shadow10: 0 2px 1px 0 value(s2), 0 0 3px 0 value(s1);
-  shadow20: 0 2px 4px 0 value(s1), 0 0 6px 0 value(s1);
-  shadow30: 0 6px 6px 0 value(s3), 0 0 18px 0 value(s1);
-  shadow40: 0 8px 8px 0 value(s1), 0 3px 24px 0 value(s4);
 }
+
+.shadow10 { box-shadow: var(--shadow10, 0 2px 1px 0 value(s2), 0 0 3px 0 value(s1)); }
```
Comment (EgoziE) [inline src/Foundation/stylable/shadows.st.css:13]: We prefer that design constants will not be mixins if the css is not complex.
Please change is to:
```
:vars {
  shadow10: var(--shadow10, 0 2px 1px 0 value(s2), 0 0 3px 0 value(s1));
}
```

and use it like before:

```
box-shadow: value(shadow10);
```

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Foundation/stylable/shadows.st.css (hunk 1 above)
Comment (dimafeldman) [reply src/Foundation/stylable/shadows.st.css]: updated. thanks!

File: src/Themes/businessDashboard/BusinessDashboard.st.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
+@st-global-custom-property --shadow10, --shadow20, --shadow30, --shadow40;
```
Comment (EgoziE) [inline src/Themes/businessDashboard/BusinessDashboard.st.css:1]: Oh sorry just now I noticed that it's global.
This means you can accidentally override those from an external css source.
I suggest to remove `@st-global-custom-property` and import those variables just like you import any other variable.

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Themes/businessDashboard/BusinessDashboard.st.css (hunk 2 above)
Comment (EgoziE) [reply src/Themes/businessDashboard/BusinessDashboard.st.css:1]: Ok so I reached a conclusion it's not possible use scoped css variables as a stylable variable value.
Bottom line, let's keep those variables as global, BUT in order to reduce any possible overrides, let's change the name from `--shadow-10` to `--wsr-shadow-10`

Comment (dimafeldman) [reply src/Themes/businessDashboard/BusinessDashboard.st.css]: updated, thanks!

**Review by EgoziE — APPROVED** [review]

**Pull request merged**